Rift-Finance/rift-finance
# Sets up husky+lint-staged

This PR sets up husky, a pre-commit hook manager, and lint-staged, which allows you to run linting only on staged files instead of on the whole repo.

Our template came with a `.huskyrc`, which does not work with the version of husky we have installed (6). As such, I ran the husky install script and setup our hooks.

To test this, you can try the following:
1) Delete a semicolon in a .sol file; when you try to commit, it should yell at you about mismatched inputs
2) Delete a semicolon in a .ts file; when you try to commit, it should fix it
